schemas/EconomicEvent.json

```diff
@@ -1,71 +1,107 @@
 {
-    "$schema": "http://json-schema.org/draft-06/schema#",
-    "id": "https://raw.githubusercontent.com/valueflows/vf-json-schema/master/schemas/EconomicEvent.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://raw.githubusercontent.com/valueflows/valueflows/master/json-schemas/EconomicEvent.json",
     "title": "EconomicEvent",
-    "description": "An inflow or outflow of an economic resource in relation to a process and/or exchange.",
+    "description": "An observed economic flow, as opposed to a flow planned to happen in the future. It could reflect a change in the quantity of an economic resource. It is also defined by its behavior in relation to the economic resource.",
     "type": "object",
     "properties": {
         "action": {
             "description": "Relates a process input or output to a verb, such as consume, produce, work, improve, etc.",
             "type": "string"
         },
-        "quantifiedAs": {
-            "description": "The quantity and unit of the event. This is the quantity that could be used to increment or decrement a resource, depending on the type of resource and type of event.",
-            "$ref": "https://raw.githubusercontent.com/valueflows/vf-json-schema/master/schemas/QuantityValue.json"
+        "resourceQuantity": {
+            "description": "The amount and unit of the economic resource counted or inventoried.",
```

Comment:
In the previous description, you added (and I reword slightly here): 
This is the quantity that could be used to increment or decrement a resource, depending on the type of resource and resource effect of action.",